heed/src/env.rs

```diff
@@ -196,6 +194,15 @@ impl EnvOpenOptions {
             }
         }
     }
+
+    pub fn close<P: AsRef<Path>>(path: P) -> Result<()> {
+        let path = canonicalize_path(path.as_ref())?;
+
+        let mut lock = OPENED_ENV.lock().unwrap();
+        let _env = lock.remove(&path);
+
+        Ok(())
+    }
```

Comment:
This close method should not be on `EnvOpenOptions`, it doesn't makes sens, I should move that to the `Env` struct itself.